inspirehep/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl

```diff
@@ -31,9 +31,11 @@
 {% macro render_record_authors(is_brief, number_of_displayed_authors=10, show_affiliations=true) %}
   {% if record.collaboration %}
     {% set collaboration_displayed = [] %}
-    {% for collaboration in record.collaboration if collaboration != None %}
-      <a href="/search?p=collaboration:'{{ collaboration }}'">{{ collaboration }}</a>
+    {% for collaboration in record.collaboration %}
+      {% if collaboration['value'] %}
```

Comment:
Does the jinja dotsyntax work in this context as well? Cause if yes, then it is better to use `collaboration.value` in case the collaboration field does not have the `"value"` key. Otherwise it may throw a keyerror.